curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-fetch-and-promises-by-building-an-fcc-authors-page/63ffa818da72e82dbe4ba0cb.md

```diff
@@ -0,0 +1,154 @@
+---
+id: 63ffa818da72e82dbe4ba0cb
+title: Step 13
+challengeType: 0
+dashedName: step-13
+---
+
+# --description--
+Now we need to show the author name and the author image. But first, let's show the author's name.
```

Comment:
We will need to remove all references to first person narrative per the [docs](https://contribute.freecodecamp.org/#/how-to-work-on-coding-challenges?id=challenge-descriptionsinstructions) here